lib/game.rb

```diff
@@ -0,0 +1,20 @@
+class Game
```

Comment:
We talked about this briefly this morning, you making two attempts at this bowling game, but I can't quite follow what the structure is (what file works on his own and which files belong together). Perhaps you can include this in a README file?